src/app/components/nav.jsx

```diff
@@ -9,6 +9,50 @@ import { IoMdGitPullRequest } from "react-icons/io";
 import Link from "next/link";
 import Switch from "@mui/material/Switch";
 
+// Sun icon component
+const SunIcon = ({ className = "" }) => (
```

Comment:
Can you create a file called icons in the components and just import it from there?